tests/test_spiderloader/__init__.py

```diff
@@ -89,3 +90,14 @@ def test_crawler_runner_loading(self):
         crawler = runner.create_crawler('spider1')
         self.assertTrue(issubclass(crawler.spidercls, scrapy.Spider))
         self.assertEqual(crawler.spidercls.name, 'spider1')
+
+    def test_bad_spider_modules_warning(self):
+
+        with warnings.catch_warnings(record=True) as w:
+            module = 'tests.test_spiderloader.test_spiders.doesnotexist'
+            settings = Settings({'SPIDER_MODULES': [module]})
+            spider_loader = SpiderLoader.from_settings(settings)
+            self.assertIn("Could not load spiders from module", str(w[0].message))
```

Comment:
Hm, I see that in #2430 the error doesn't have a full traceback pointing to the module being imported (traceback is unrolled to the importlib and then to scrapy code); here it is the same. Could you please check that there is no a good traceback in this case even in Python 3? Is it the only information available for this ImportError? Sorry for being so picky!